reflex/components/forms/checkbox.py

```diff
@@ -50,6 +50,9 @@ class Checkbox(ChakraComponent):
     # The name of the input field in a checkbox (Useful for form submission).
     name: Var[str]
 
+    # The value of the input field when checked (use is_checked prop for a bool)
```

Comment:
Should we call this something like `checked_value`? It may be a bit confusing because for other components `value` is used as the name for the controlled input